telegram/_bot.py

```diff
@@ -5020,19 +5029,26 @@ async def send_invoice(
             photo_width (:obj:`int`, optional): Photo width.
             photo_height (:obj:`int`, optional): Photo height.
             need_name (:obj:`bool`, optional): Pass :obj:`True`, if you require the user's full
-                name to complete the order.
+                name to complete the order. Ignored for payments in `Telegram Stars
+                <https://t.me/BotNews/90>`_.
```

Comment:
rst substitation? maybe at least for "`Telegram Stars <https://t.me/BotNews/90>`_"?